.github/workflows/DeployWiki.yml

```
@@ -0,0 +1,26 @@
+# https://github.com/marketplace/actions/github-wiki-action
+name: Deploy Wiki
+
+on:
+  push:
+    paths:
+      # Trigger only when wiki directory changes
+      - "wiki/**"
+    branches:
+      # And only on master branch
```

O comentário está errado
```suggestion
      # And only on main branch
```